Fix Extra Spacing In All Gnome Shell Menus

TheWiseNoob opened the pull request:
This fixes the extra spacing/margins added to menu items in all gnome panel menus bug mentioned in issue #483. The CSS was too general, so it targeted all gnome panel menu items.

 It also removed unneeded left space added to clipboard history menu items since only a right margin was needed.

**Before:**
![image](https://github.com/user-attachments/assets/b1118fd6-1749-4dc5-a60e-e4dee1fa8d3e)
![image](https://github.com/user-attachments/assets/8ed1a270-c2db-4168-a423-4a53aa20fc73)


**After:** 
![image](https://github.com/user-attachments/assets/3093ba90-d1da-4774-8f21-94d1626357ba)
![image](https://github.com/user-attachments/assets/c5f869c0-9c5e-46d0-9ff1-172cf491f276)

Tudmotu commented:
Thank you!
Will publish to e.g.o later today 🙏

Tudmotu commented:
Published to e.g.o, v64.
Really appreciate you taking care of this issue :pray:

TheWiseNoob commented:
Glad to help!

Thanks for the quick merge and publish to EGO!

For cleanup after you merged, I deleted this branch and deleted my fork as recommended by GitHub.
